styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: content-box;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.menu {
+  width: 18px;
+  height: 20px;
+}
+
+.top-menu {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 25px;
+  width: 375px;
+  height: 12px;
+}
+
+.toolbarlogo {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  color: #fff;
+  background-color: #3c3a39;
+  height: 20px;
+}
+
+.welcomeinfo {
+  display: flex;
+  flex-direction: column;
+  align-content: stretch;
+  width: 375px;
+  padding: 25px;
+  background-image: url(./icons/flair.png);
+  background-repeat: no-repeat;
+  background-position: 278px 220px;
+  height: 600px;
+}
+
+.title {
+  font-family: 'Crete Round', serif;
+  font-size: 56px;
+  color: #ff6b00;
+}
+
+.subtitle {
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  color: #fff5e1;
+}
+
+.paragraph {
+  font-family: 'Poppins', sans-serif;
+  color: #fff;
+}
+
+.contact-bar {
+  display: flex;
+  margin-top: -180px;
+  margin-left: 80px;
+}
```

Comment:
[Required Changes]
- [ ] Please remove the margin-left style here so the social icons are placed on the left. That way your design will match the one given in the Figma design.

### Figma Design
![image](https://user-images.githubusercontent.com/50941074/158849622-964f0c70-ad92-4ae1-9555-03b101c25c7c.png)

### Your Design
![image](https://user-images.githubusercontent.com/50941074/158849932-61448a35-b745-4fe7-81ae-525a66be4914.png)

Reply:
Just finished this change and will submit it for a new review. Thank you for your help, Amadu!